### PR TITLE
fix: TextArea showCount style while using with Form

### PR DESCRIPTION
**Pull request by AlanCutFlim**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
#28123 

### 💡 Background and solution
Before:
image
After:
image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Fix TextArea showCount style while using with Form     |
| 🇨🇳 Chinese |     修复 TextArea 结合 Form 使用时的字数提示样式问题     |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/input/style/index.less [hunk 1]
```diff
@@ -46,6 +46,9 @@
 
   &-textarea {
     &-show-count::after {
+      position: absolute;
+      right: 0;
+      bottom: -22px;
```
Comment (afc163) [inline components/input/style/index.less:51]: 在 Form 中单独修改吧，否则会有问题：

https://preview-28130-ant-design.surge.sh/components/input-cn/#components-input-demo-textarea-show-count

Comment (AlanCutFlim) [reply components/input/style/index.less]: Done

**Comment by AlanCutFlim** [comment]
@afc163 请问 argos ci 里的错误是什么意思？
image

File: components/input/style/index.less [hunk 2]
```diff
@@ -47,6 +47,7 @@
   &-textarea {
     &-show-count::after {
       display: block;
+      width: 100%;
```
Comment (afc163) [inline components/input/style/index.less:50]: 加这个的目的是？

Comment (AlanCutFlim) [reply components/input/style/index.less:50]: > 加这个的目的是？

原来宽度是占满一行的，absolute 脱离文档流之后宽度就是内容的宽度了，此时 rtl 下就不会居左。加这个就是为了用原来的 `text-align: left` 来处理 rtl 时的情况。

Comment (afc163) [reply components/input/style/index.less:50]: 如果针对 absolute 的情况就放到 form 的样式去，不影响原来的样式。

**Comment by shaodahong** [comment]
> @afc163 请问 argos ci 里的错误是什么意思？

已修复，可以 rebase 下 master

File: components/input/style/index.less (hunk 2 above)
Comment (AlanCutFlim) [reply components/input/style/index.less]: Done

**Pull request merged**